test/unit/testIhtPaper.js

```diff
@@ -14,32 +14,45 @@ describe('IhtPaper', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
-
-        it('should return the ctx with the deceased married status and the screening_question feature toggle', (done) => {
-            ctx = {
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
```

Comment:
Please update the description for this test as it is for iht paper but it is referring to deceased address.